src/App/Pages/Settings/SettingsPage/SettingsPageListItem.cs

```diff
@@ -22,5 +23,41 @@ public class SettingsPageListItem : ISettingsPageListItem
         public Color SubLabelColor => SubLabelTextEnabled ?
             ThemeManager.GetResourceColor("SuccessColor") :
             ThemeManager.GetResourceColor("MutedColor");
+        public string AutomationId
+        {
+            get
+            {
+                if (!UseFrame)
+                {
+                    var idText = new CultureInfo("en-US", false)
+                    .TextInfo
+                    .ToTitleCase(Name)
+                    .Replace(" ", String.Empty)
+                    .Replace("-", String.Empty);
+                    return $"{idText}Cell";
+                }
+                else
+                {
+                    return "EnabledPolicyCell";
+                }
+            }
+        }
```

Comment:
♻️ I see that the logic is repeated in other places (like in the `AutomationIdSettingStatus` property). So extract this to an extension method or helper class that you can reuse. 
You can have a class named `AutomationIdsHelper` that could have this kind of stuff, like a static method `ToEnglishTitleCase(string)` which could hold this logic.
Also, you can add an add suffix static method like `AddSuffixFor(string, SuffixType)` with `SuffixType` being an enum of the different things to suffix like `...Cell` and inside add the suffix to the correct component and with that you have less error prone code because of hardcoded strings. 
Then for this scenario you could have something like this:
```c#
public string AutomationId
{
    get
    {
        return AutomationIdsHelper.AddSuffixFor( 
                    UseFrame ? "EnabledPolicy" 
                             : AutomationIdsHelper.ToEnglishTitleCase(Name)
                    , SuffixType.Cell);
    }
}
``` 
---
⛏️ Also, given that there are more than one `AutomationId...` properties, give a more descriptive name to this one to know how it differentiate from `AutomationIdSettingStatus`

Reply:
I'm still feeling like a foreigner in this repository, and that's why I choose to add those similar methods instead of adding new files to the project.

What is the best location to store this helper class? I'm thinking about creating a new `AutomationHelpers` inside `App/Utilities` but I'd like to read your thoughts 🙌

Reply:
`App/Utilities` is fine but I'd add an `Automation` folder in there so you can add anything related to that there. Therefore the location would be `App/Utilities/Automation`